README.md

```diff
@@ -45,9 +45,29 @@ in those directories.  These subdirectories are:
 
 ## Dependencies
 
+### Build system
+
+To build Toit and its dependencies the build host requires;
+
+* [GNU Make](https://www.gnu.org/software/make/)
+* [CMake >= 3.13.3](https://cmake.org/)
+* [Ninja](https://ninja-build.org/)
+* [GCC](https://gcc.gnu.org/)
+* [Go](https://go.dev/),
```

Comment:
Drop the comma at the end (for consistency with the other lists).

Reply:
Ahh sorry, had a rebase with this fixed which went stale while waiting for CLA stuff to resolve.